integration-tests/docker/Dockerfile

```diff
@@ -1,15 +1,4 @@
-FROM ubuntu:14.04
-
-# Java 8
-RUN apt-get update \
-      && apt-get install -y software-properties-common \
-      && apt-add-repository -y ppa:webupd8team/java \
-      && apt-get purge --auto-remove -y software-properties-common \
-      && apt-get update \
-      && echo oracle-java-8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
-      && apt-get install -y oracle-java8-installer \
-      && apt-get install -y oracle-java8-set-default \
-      && rm -rf /var/cache/oracle-jdk8-installer
+FROM jonweiimply/ubuntu-j8
```

Comment:
This line certainly needs some comments, ideally describing the image, linking to the Dockerfile that produced the image, and linking to this issue. It should also be temporary; even if we want to use our own Docker base image long term, it should be something in an organization repo rather than a personal repo.

Reply:
Added a comment on the temporary nature of the change and a link to the dockerhub page for the image, which has a description of the commands used to build it